experiment/resources/schemas/dbscripts/postgresql/exp-22.001-22.002.sql

```diff
@@ -0,0 +1 @@
+UPDATE exp.Protocol SET Status = 'Active' WHERE ApplicationType = 'ExperimentRun';
```

Comment:
We don't want to stomp over any deactivated protocols, do we?

```suggestion
UPDATE exp.Protocol SET Status = 'Active' WHERE ApplicationType = 'ExperimentRun' WHERE Status IS NULL;
```